types/types.go

```diff
@@ -83,13 +84,13 @@ func (status HealthStatus) String() string {
 	return "unknown"
 }
 
-func GetInstanceStatusTag(svcContainer SvcContainer, healthy HealthStatus, running bool, exitCode int) map[string]interface{} {
-	return map[string]interface{}{
+func GetInstanceStatusTag(svcContainer SvcContainer, healthy HealthStatus, running bool, exitCode int) map[string]string {
+	return map[string]string{
```

Comment:
What's the rational behind this change? Will it cause issues upstream for us?

Reply:
this is with regards to otel version upgrade. otel tags no longer accept a map of string-interface